app/assets/stylesheets/public_view_shop.sass

```diff
@@ -0,0 +1,65 @@
+$semi-white: #FF9495
+
+#public-view-shop-header
+  // background-color: $white
+  .card-block
+    background-color: $white
+
+    .text-xs-center
+
+      .card-title
+        margin-top: -60px
+        margin-bottom: 40px
+        padding-left: 70px
+
+      .btn
+        background-image: linear-gradient(to right, $blue 0%, #187CB5 100%)
```

Comment:
Could we also get these linear-gradients in a class in the utilities and then you can just use `@include` to include that style. It's more centralised.